content/en/tracing/guide/inferred-service-opt-in.md

```diff
@@ -16,30 +16,34 @@ further_reading:
+With the new inferred entities experience, filter [Service Catalog][3] entries by entity type (database, queue, third-party API, etc), and better visualize service dependencies from the new [Service Page dependency map](#service-page-dependency-map) and other places in the APM product experience.
 
-To determine the names and types of the inferred entities, Datadog uses span attributes. Inferred external APIs use the default naming scheme `net.peer.name`. For example, `api.stripe.com`, `api.twilio.com`, `us6.api.mailchimp.com`. Inferred databases use the default naming scheme `db.instance`.
+To determine the names and types of the inferred service dependencies, Datadog uses standard span attributes and maps them to `peer.*` attributes - find the full list of `peer.*` attributes [here](#inferred-service-dependencies-nomemclature) -. Inferred external APIs use the default naming scheme `net.peer.name`. For example, `api.stripe.com`, `api.twilio.com`, `us6.api.mailchimp.com`. Inferred databases use the default naming scheme `db.instance`.
 
 If you're using the Go, Java, NodeJS, PHP, .NET, or Ruby tracer, you can customize the default names for inferred entities. 
 
-**Note:** If you configure monitors, dashboards, or notebooks for a given inferred service during the beta, you may need to update them if the naming scheme changes.
+**Note:** If you configure monitors, dashboards, or notebooks for a given inferred service during the beta, you may need to update them if the naming scheme changes. Read more about migration steps in the [opt-in instructions](#opt-in).
 
-## Dependency map
+### Service page Dependency map
 
 Use the dependency map to visualize service-to-service communication and gain insight into system components such as databases, queues, and third-party dependencies. You can group dependencies by type and filter by Requests, Latency, or Errors to identify slow or failing connections.
 
-{{< img src="tracing/services/service_page/dependencies.png" alt="The dependency section" style="width:100%;">}}
+{{< img src="tracing/services/service_page/dependencies.png" alt="Service page service dependency map" style="width:100%;">}}
 
 ## Opt in
 
-To opt in, you must adjust your Datadog Agent and APM Tracer configurations. Check the [Global default service naming migration](#global-default-service-naming-migration), to see if you need to take any migration actions.
+<div class="alert alert-warning">Only go through migration steps once Datadog support confirmed the feature is enabled for you on the Datadog side.</div>
+
+To opt in, you must adjust:
+- your [Datadog Agent](#datadog-agent-configuration) (or [OpenTelemetry collector](#opentelemetry-collector)) configuration
+- your [APM tracing libraries](#apm-tracing-libary-configuration) configuration
```

Comment:
This is optional but recommended. If a customer doesn't want to take on the breaking changes, the experience still works but not as well.